pkg/cmd/recoveryapiserver/lib.go

```diff
@@ -0,0 +1,35 @@
+package recoveryapiserver
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+)
+
+type Options struct {
```

Comment:
this are shared options, right? Would better call it like that. Was confused when reading the code.